File: nodejs/7 kyu/172.js
// https://www.codewars.com/kata/596f6385e7cd727fff0000d6/train/javascript

// Passed

function avgArray(arr) {
  let averages = [];
  for (idx = 0; idx < arr[0].length; idx++) {
    let idxSum = 0;
    for (valArr of arr) {
      idxSum += valArr[idx];
    }
    let average = idxSum / arr.length;
    averages.push(average);
  }

  return averages;
}

const output = avgArray([
  [1, 2, 3, 4],
  [5, 6, 7, 8],
]);
console.log(output);

File: python/7 kyu/172.py
# https://www.codewars.com/kata/541629460b198da04e000bb9/train/python

# Passed

def last(*args):
    if len(args) == 1:
        elem = args[0]
        if type(elem) == int:
            return elem
        
        return elem[-1]
    
    return args[-1]

output = last(123, [4, 5, 6])
print(output)